.github/workflows/docker_image.yml

```diff
@@ -173,6 +171,7 @@ jobs:
         uses: docker/build-push-action@v6
         with:
           context: .
+          build-args: "JAVA_VERSION=1.8.0"
```

Comment:
@PHILO-HE see if the config works. I didn't find examples

Reply:
This should be correct. We can remove `""`. See reference: https://medium.com/@nynptel/building-and-pushing-docker-images-in-the-github-action-workflow-179e6a0411bc